database/README.md

```diff
@@ -51,27 +51,25 @@ The CRIS data in our database consists of four record types:
+- `crashes`: a unified version of each record which combines the values in `crashes_cris` plus any edits created by Vision zero staff through the Vision Zero Editor web app. This table functions as the final source of truth, where VZ edits take precedence over CRIS values.`
 
 As pictured in the diagram below, the typical data flow for a crash record is as follows:
 
 1. A new record is inserted into the `crashes_cris` table through the [CRIS import ETL](../etl/cris_import/README.md).
-2. On insert into `crashes_cris`, an "empty" copy of the record is inserted into the `crashes_edits` table. The record is inserted into the `crashes_edits` table with null values in every column except the `id`, which has a foreign key constraint referencing the `crashes_cris.id` column.
-3. At the same time, a complete copy of the record is inserted into the `crashes` table.
-4. A Vision Zero Editor user may update crash records by updating rows in the `crashes_edits` table. When an update is received, a trigger function coalesces each value in the `crashes_edits` table with the corresponding value in the `crashes_cris` table. The resulting record—which contains the original CRIS-provided values plus any edit values made through user edits—is applied as an update to the corresponding record in the `crashes` table.
-5. Similarly, when an existing `crashes_cris` record is updated through the CRIS import ETL, the updated record is coalesced against the corresponding row in the `crashes_edits` table, and the result is saved in the `crashes` table.
-6. Finally, once a record is updated in the `crashes` table, additional trigger functions apply various business rules and enrich the row with spatial attributes based on its location. These trigger functions are reserved for values that require heavy computation—additional business rules can be applied through table views.
+2. On insert into `crashes_cris`, a complete copy of the record is inserted into the `crashes` table.
+3. A Vision Zero Editor user may update crash records by updating rows in the `crashes` table.
+4. When an existing `crashes_cris` record is updated through the CRIS import ETL, a trigger function compares the old value in the `crashes_cris` table to the value in the unified `crashes` table and if they are the same, meaning there have been no VZ edits, the `crashes` value is also updated with the new value. If there has been a VZ edit, only the `crashes_cris` table is updated.
+5. Once a record is updated in the `crashes` table, additional trigger functions apply various business rules and enrich the row with spatial attributes based on its location. These trigger functions are reserved for values that require heavy computation—additional business rules can be applied through table views.
 
 ![CRIS editing model](../docs/images/cris_data_model.png)
-_The "layered" editing environment of the Vision Zero Database_
+_The "two layer" editing environment of the Vision Zero Database_
```

Comment:
fwiw, Xavier is at least one person who considers the DB views to be another layer in our model. i don't mind keeping the generic "layered" term around 🤷